solidity/contracts/isms/warp-route/RateLimitedIsm.sol

```diff
@@ -42,7 +52,12 @@ contract RateLimitedIsm is
     function verify(
         bytes calldata,
         bytes calldata _message
-    ) external validateMessageOnce(_message) returns (bool) {
+    )
+        external
```

Comment:
This seems like a good time to the mermaid diagrams and how atomicity with process is enforced

Reply:
opened https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4637